src/_help/customization-options/operations-navigation.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Operations and navigation
+---
+
+- TOC
+{:toc}
+
+Bump.sh offers several methods for sorting your API operations to provide a smooth and coherent reading structure, based on either the tags or operation names and URLs.
+
+Supported API specifications allow for several types of requests:
+
+- Endpoints are typically URLs that point to a specific resource on an API server. In OpenAPI, endpoints are stored under the `paths` resource.
+- Webhooks are requests generated by the API and sent to an external server after a specific event has occurred. In OpenAPI 3.1, they are stored under the `webhooks` resource.
+
+> If first-level field `tags` are present at the [root of your OpenAPI document object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#openapi-object), Bump.sh will use `Group by tags` as a default documentation generation behaviour.
```

Comment:
This should be moved in a `## Automatically` section, explaining the default (= if "Automatically" is selected) mode of grouping.